br/pkg/streamhelper/advancer_env.go

```diff
@@ -137,3 +146,33 @@ type StreamMeta interface {
 	// ClearV3GlobalCheckpointForTask clears the global checkpoint to the meta store.
 	ClearV3GlobalCheckpointForTask(ctx context.Context, taskName string) error
 }
+
+var _ tikv.RegionLockResolver = &AdvancerLockResolver{}
+
+type AdvancerLockResolver struct {
+	store tikv.Storage
+	*tikv.BaseRegionLockResolver
+}
+
+func newAdvancerLockResolver(store tikv.Storage) *AdvancerLockResolver {
+	return &AdvancerLockResolver{
+		store:                  store,
```

Comment:
...Then, it seems the `store` can be removed and replaced with `.BaseRegionLockResolver.GetStore()`